frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/LinkOptions/LinkedEntityPicker/LinkedEntityPicker.tsx

```diff
@@ -233,6 +237,12 @@ function LinkedEntityPicker({
     ],
   );
 
+  const dashboard = useSelector(getDashboard);
```

Comment:
Maybe worth naming this `currentDashboard`? I know the other dashboard is called `targetDashboard` but considering before this change, dashboard meant the target one it could make sense

Reply:
We use `dashboard` pretty much as a value returned from `getDashboard` everywhere, so I'd like to keep that consistency.

Also, usually with `currentSomething` it should be used to differentiate from other Something e.g. `previousDashboard` vs `currentDashboard` if we need to compare the 2 versions for some reasons. But that's not the case here either.